src/pages/ReimbursementAccount/ContinueBankAccountSetup.js

```diff
@@ -43,6 +43,7 @@ function ContinueBankAccountSetup(props) {
                 title={props.translate('workspace.common.connectBankAccount')}
                 subtitle={props.policyName}
                 shouldShowGetAssistanceButton
+                onBackButtonPress={props.onBackButtonPress}
```

Comment:
I think the `onBackButtonPress` prop is missing in `propTypes` of this component